package com.example.medicineordering.entity;

package com.pharmacy.entity;

import jakarta.persistence.*;

import lombok.*;

@Entity
@Table(name = "cart_items")

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class CartItem {

    @Id
    @GeneratedValue(strategy =
            GenerationType.IDENTITY)

    private Long id;

    @ManyToOne
    @JoinColumn(name = "cart_id")

    private Cart cart;

    @ManyToOne
    @JoinColumn(name = "medicine_id")

    private Medicine medicine;

    private Integer quantity;

    private Double subtotal;
}